modules/exploits/apple_ios/browser/webkit_trident.rb

```diff
@@ -0,0 +1,299 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
```

Comment:
Should probably be `Average` or `Good` ranking if there's a 20% chance of crash.